po/keys.pot

```diff
@@ -4179,1122 +4210,1141 @@ msgid "specify an organization"
 msgstr ""
 
-#: src/subscription_manager/managercli.py:2163
+#: src/subscription_manager/managercli.py:2178
 msgid "Error: The 'release' command is not supported by the server."
 msgstr ""
 
-#: src/subscription_manager/managercli.py:2172
+#: src/subscription_manager/managercli.py:2187
 #, python-format
 msgid "Release: %s"
 msgstr ""
 
-#: src/subscription_manager/managercli.py:2174
+#: src/subscription_manager/managercli.py:2189
 msgid "Release not set"
 msgstr ""
 
-#: src/subscription_manager/managercli.py:2194
+#: src/subscription_manager/managercli.py:2209
 msgid "Release preference has been unset"
 msgstr ""
 
-#: src/subscription_manager/managercli.py:2210
+#: src/subscription_manager/managercli.py:2225
 #, python-format
 msgid "No releases match '%s'.  Consult 'release --list' for a full listing."
 msgstr ""
 
-#: src/subscription_manager/managercli.py:2214
+#: src/subscription_manager/managercli.py:2229
 #, python-format
 msgid "Release set to: %s"
 msgstr ""
 
-#: src/subscription_manager/managercli.py:2224
+#: src/subscription_manager/managercli.py:2239
 msgid "No release versions available, please check subscriptions."
 msgstr ""
 
-#: src/subscription_manager/managercli.py:2227
+#: src/subscription_manager/managercli.py:2242
 msgid "Available Releases"
 msgstr ""
 
-#: src/subscription_manager/managercli.py:2248
+#: src/subscription_manager/managercli.py:2263
 msgid "The ID of the pool to attach (can be specified more than once)"
 msgstr ""
 
-#: src/subscription_manager/managercli.py:2250
+#: src/subscription_manager/managercli.py:2265
 msgid "Number of subscriptions to attach. May not be used with an auto-attach."
 msgstr ""
 
-#: src/subscription_manager/managercli.py:2252
+#: src/subscription_manager/managercli.py:2267
 msgid ""
 "Automatically attach compatible subscriptions to this system. This is the "
 "default action."
 msgstr ""
 
-#: src/subscription_manager/managercli.py:2254
+#: src/subscription_manager/managercli.py:2270
 msgid ""
 "Automatically attach only subscriptions matching the specified service "
 "level; only used with --auto"
 msgstr ""
 
-#: src/subscription_manager/managercli.py:2256
+#: src/subscription_manager/managercli.py:2273
 msgid ""
 "A file from which to read pool IDs. If a hyphen is provided, pool IDs will "
 "be read from stdin."
 msgstr ""
 
-#: src/subscription_manager/managercli.py:2259
+#: src/subscription_manager/managercli.py:2277
 msgid "All installed products are covered by valid entitlements."
 msgstr ""
 
-#: src/subscription_manager/managercli.py:2260
+#: src/subscription_manager/managercli.py:2278
 msgid "No need to update subscriptions at this time."
 msgstr ""
 
-#: src/subscription_manager/managercli.py:2271
-msgid "Attach a specified subscription to the registered system"
+#: src/subscription_manager/managercli.py:2290
+msgid ""
+"Attach a specified subscription to the registered system, when system does "
+"not use Simple Content Access mode"
```

Comment:
when the system does

Reply:
This is a typo in the code - what do you think about fixing later on?